### PR TITLE
update SpnegoHelper javadoc

### LINKED ISSUE #25655: Revise SpnegoHelper.buildS4U2ProxyAuthorizationUsingS4U2Self javadoc
Update javadoc for SpnegoHelper.buildS4U2ProxyAuthorizationUsingS4U2Self javadoc
1. change firstServiceSpn to first service SPN
2. say jaasLoginContextEntry is not currently supported. (search for right phrase: this parameter is not supported)

code: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/websphere/security/s4u2proxy/SpnegoHelper.java#L92

the update will be reflected in this link to the javadoc: https://www.ibm.com/docs/en/was-liberty/nd?topic=SSAW57_liberty/com.ibm.websphere.javadoc.liberty.doc/com.ibm.websphere.appserver.api.constrainedDelegation_1.0-javadoc/com/ibm/websphere/security/s4u2proxy/SpnegoHelper.htm

### PR DESCRIPTION
**Pull request by karel-harjono**
fixes #25655
change firstServiceSPN to first service SPN and note for not used param.

### CODE REVIEW

**Comment by karel-harjono** [comment]
#build

File: dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/websphere/security/s4u2proxy/SpnegoHelper.java [hunk 1]
```diff
@@ -72,19 +72,18 @@ public class SpnegoHelper {
...
-     * The method will use that credential to request a SPNEGO token for a targetServiceSpn (SPN) for the target service system.
+     * The method will use that credential to request a SPNEGO token for a target service principal name  for the target service system.
      * If the system property java.security.krb5.conf has not been set by the run time, you need to set the java.security.krb5.conf that point to your Kerberos configuration file.
      *
-     * @param upn - UserPrincipalName of the user for which the SPNEGO token will be generated.
-     * @param targetServiceSpn - ServicePrincipalName of system for which SPNEGO token will be targeted.
-     * @param lifetime - Lifetime for the context, for example GSSCredential.INDEFINITE_LIFETIME
+     * @param upn – User principal name of the user for which the SPNEGO token will be generated.
+     * @param targetServiceSpn – Service principal name of a system for which SPNEGO token will be targeted.
+     * @param lifetime – Life time for the context, for example GSSCredential.INDEFINITE_LIFETIME.
```
Comment (Zech-Hein) [inline dev/com.ibm.ws.security.kerberos.java8/src/com/ibm/websphere/security/s4u2proxy/SpnegoHelper.java]: lifetime is actually one word, so we should keep it how it was before.
```suggestion
     * @param lifetime – Lifetime for the context, for example GSSCredential.INDEFINITE_LIFETIME.
```

**Comment by karel-harjono** [comment]
#libby

**Review by Zech-Hein — APPROVED** [review]
LGTM!

**Pull request merged**